nautobot/dcim/migrations/0051_interface_redundancy_group_nullable_status.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations, models
+from nautobot.extras.utils import fixup_null_statuses
+
+
+def migrate_null_statuses(apps, schema):
+    Status = apps.get_model("extras", "Status")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    InterfaceRedundancyGroup = apps.get_model("dcim", "interfaceredundancygroup")
+    interface_redundancy_group_ct = ContentType.objects.get_for_model(InterfaceRedundancyGroup)
+    fixup_null_statuses(
+        model=InterfaceRedundancyGroup, model_contenttype=interface_redundancy_group_ct, status_model=Status
+    )
+
+
+def copy_created_to_created_datetimefield(apps, schema):
+    InterfaceRedundancyGroupAssociation = apps.get_model("dcim", "interfaceredundancygroupassociation")
+    InterfaceRedundancyGroupAssociation.objects.update(created_datetimefield=models.F("created"))
+
+
+def revert_copy_created_to_created_datetimefield(apps, schema):
+    InterfaceRedundancyGroupAssociation = apps.get_model("dcim", "interfaceredundancygroupassociation")
+    InterfaceRedundancyGroupAssociation.objects.update(created_datetimefield=None)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("dcim", "0022_interface_redundancy_group"),
```

Comment:
We probably won't need this line here since we are already running it before 0050

Reply:
This is the actual migration that this one depends on. The 0050 dependency is just to prevent the multiple leaf node problem that Django doesn't like. I'd rather keep these as-is